NEWS.md

```diff
@@ -8,6 +8,7 @@
 
 ## Model changes
 
+- `plot.dist_spec` now uses color instead of line types to display pmfs vs cmfs. By @jamesmbaazam in #788 and reviewed by <REVIEWER>.
```

Comment:
```suggestion
- `plot.dist_spec` now uses color instead of line types to display pmfs vs cmfs. By @jamesmbaazam in #788 and reviewed by @sbfnk .
```